features/speech-recognition-grammar.yml

```diff
@@ -0,0 +1,23 @@
+name: Speech recognition grammar
```

Comment:
SpeechGrammar and SpeechGrammarList were marked a deprecated in BCD because they were removed from the spec entirely.

We can't have deprecated keys inside of a published, non discouraged, feature. So I resorted to moving these keys out of speech-recognition and into a new, discouraged, speech-recognition-grammar feature.

I'm a little concerned with this. This isn't the first time I stumble upon this limitation and I feel like features should be allowed to have deprecated keys in them. Sometimes they'll need to be there because there isn't any other feature for them, and will just get deleted after a while.

Similar to `compute_from`, I feel like we need a `ignore` field to house the keys that, logically, belong to a feature, but which shouldn't impact its status.

Reply:
I think it makes sense to split this out like you did- it better communicates that this subset is no longer a part of the feature.